contracts/v0.2/README.md

```diff
@@ -39,3 +39,130 @@ cp .env.example .env
 ```shell
 forge script DeployFull --broadcast --rpc-url [RPC]
 ```
+
+## Migration Examples
+
+### Deploy `Feed` & `FeedProxy` Contracts
+
+```
+{
+  "Feed": {
+    "deploy": {
+      "submitter": "0xa195bE68Bd37EBFfB056279Dc3d236fAa6F23670",
+      "count": 2,
+      "feed": [
+        { "decimals": 8, "description": "ADA-USDT" },
+        { "decimals": 8, "description": "ATOM-USDT" }
+      ]
+    }
+  }
+}
+```
+
+### Update Submitter of `Feed` Contract
+
+```
+{
+  "Feed": {
+    "address": "0x1ac6cd893eddb6cac15e5a9fc549335b8b449015",
+    "updateSubmitter": {
+      "submitter": "0xa195bE68Bd37EBFfB056279Dc3d236fAa6F23670",
+      "count": 2,
+      "feeds": [
+        "0xc765f5ed9abb26349054020feea04f955a5cb1ec",
+        "0x9bb8f7b9f08ecc75aba62ba25d7b3f46fce79745"
+      ]
+    }
+  }
+}
+```
+
+### Deploy `FeedRouter`
+
+```
+{
+  "FeedRouter": {
+    "deploy": {}
+  }
+}
+
+```
```

Comment:
@nick-bisonai Yes, it is possible. It would be basically combination of the config above and below.

```
{
  "FeedRouter": {
    "deploy": {},
    "updateProxyBulk": {
      "count": 2,
      "proxies": [
        {
          "feedName": "BTC-USDT",
          "proxyAddress": "0x50c23983ea26f30d368da5b257001ee3ddf9a539"
        },
        {
          "feedName": "KLAY-USDT",
          "proxyAddress": "0xd07bd0bcd3a8fa1087430b1be457e05c4a412a4b"
        }
      ]
    }
  }
}
```